### ISSUE TITLE
Plant UML Diagram

### ISSUE THREAD

**Issue by tsalinger** [qjebbs/vscode-plantuml#107]
>This issue has been moved from [Microsoft/vscode/](https://github.com/Microsoft/vscode/issues/46086)
Original author is <a href="/JannLee">
      <img class="avatar rounded-1" height="44" width="44" alt="@JannLee" src="https://avatars1.githubusercontent.com/u/12993759?s=88&amp;v=4">
    </a>@JannLee
---
Issue Type: <b>Bug</b>

After update to VisualStudio Code 1.21.1

I cannot show plant uml diagram in preview panel.

How can I solve it?

VS Code version: Code 1.21.1 (79b44aa704ce542d8ca4a3cc44cfca566e7720f1, 2018-03-14T14:46:47.128Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 1800)|
|Memory (System)|7.84GB (4.44GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe D:\Desktop\프로젝트 회의.md|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
plantuml|jeb|2.5.5
markdown-preview-enhanced|shd|0.3.4

</details>
Reproduces only with extensions

---

**Comment by wegascee** [comment]
Recently, I installed VisualStudio Code 1.21.1. I used the Windows Power Shell Command to install all dependencies. Afterwards I installed the PlantUML extension.

Now I'm asking how I could create a new diagram. If I take an example and paste it into a new file, nothing happens (no syntax highlighting, no preview). After saving the file as `wsd` I now have syntax highlighting, but no preview window appears.

Seems the I have the same problem here ...
